### DISCUSSION TITLE
Keep source formatting

### DISCUSSION THREAD

**Discussion by jkefc** [apostrophecms/sanitize-html#653] [Q&A]
Hey, 
I tried using sanitize-html with content saved with CMS but there is a conflict as CKEditor used in the CMS puts specific formatting of the html source code, which is different from sanitize-html. is it possible to retain specific formatting (new lines, indentations,etc) in the output?

**Comment by boutell** [comment]
Whitespace formatting in HTML, generally speaking, doesn't matter. Can you be more specific about how this is an issue in practice?
